plugins/torrent/deluge_

```diff
@@ -0,0 +1,399 @@
+#! /usr/bin/env python2
+
+"""=cut
+=head1 NAME
+
+deluge_ - Munin wildcard plugin to monitor Deluge torrent client
+
+=head1 REQUIREMENTS
+
+ - Python2.5+ (Deluge itself won't work with python3)
+ - Deluge
+
+This plugin also uses
+ - deluge.ui.client
+ - deluge.log
+ - twisted
+These modules are required by Deluge itself.
+
+=head1 INSTALLATION
+
+This plugin has 3 modes :
+ - connections : monitors the number of connections
+ - bandwidth : monitors the bandwidth (up, up overhead, down, down overhead)
+ - states : monitors the torrents' states
+
+To use one of these modes, link the this plugin as 'deluge_<mode>'
+For example :
+ln -s /path/to/deluge_ /etc/munin/plugins/deluge_connections
+
+=head1 CONFIGURATION
+
+Use your "/etc/munin/plugin-conf.d/munin-node" to configure this plugin.
+You must at least add :
+    [deluge_*]
+   user <user_with_access_to_deluge>
+   env.HOME <path_to_deluge_user_home>
+
+By default, this plugin will try to access the deluge daemon with the following
+settings :
+   host 127.0.0.1
+   port 58846
+   no username
+   no password
+
+You can change these settings in "plugin-conf.d/munin-node" :
+    [deluge_*]
+   user <user_with_access_to_deluge>
+   env.HOME <path_to_deluge_user_home>
+   env.host 127.0.0.1
+   env.port 58846
+   env.username user
+   env.password pass
+
+By default, deluge configuration files will be searched under $XDG_CONFIG_HOME,
+which is by default set to $HOME/.config
+Setting env.HOME allows this default to work. However, you can also explicitly
+set the env.XDG_CONFIG_HOME if needed.
+
+=head1 INTERPRETATION
+
+=head2 connections
+
+In the "connections" mode, this plugin shows a graph with the number of
+connections
+
+=head2 bandwidth
+
+In the "bandwidth" mode, this plugin show graphs for the download and upload
+bandwidths.
+Each of them has "payload" and "overhead" value.
+ - with positive values : the upload values
+ - with negative values : the download values
+
+=head2 states
+
+In the "states" mode, this plugin shows the number of torrents in each state :
+Downloading, Seeding, Paused, Error, Queued, Checking, Other
+
+=head1 MAGIC MARKERS
+
+  #%# family=auto
+  #%# capabilities=autoconf suggest
+
+=head1 VERSION
+
+1.0.0
+
+=head1 AUTHOR
+
+Neraud (https://github.com/Neraud)
+
+=head1 LICENSE
+
+GPLv2
+
+=cut"""
+
+
+import logging
+import os
+import string
+import sys
+
+try:
+    from deluge.log import setupLogger
+    from deluge.ui.client import client
+    from twisted.internet import reactor, defer
+    setupLogger()
+except (ImportError, NameError):
```

Comment:
As I'm not used to writing Python code, I tried the 2 following tests : 

`from [unknown.package] import [submodule]`
-> `ImportError`

`from [existing.package] import doestNotExist`
-> `NameError`

I assume it's rare to have a stable module renaming a submodule.

Is it common practice to only check for ImportError and assume that if/when a submodule is renamed, everything breaks down ?